lib/beaker-puppet/install_utils/puppet5.rb

```diff
@@ -27,7 +27,29 @@ def fetch_build_details(sha_yaml_url)
         return sha_yaml_folder_url, file_hash[:platform_data]
       end
 
-      # gets the artifact & repo_config URLs for this host in the build
+      # Get the host's packaging platform, based on beaker-hostgenerator's
+      # osinfo hash and the environment. Set ENV['BEAKER_PACKAGING_PLATFORMS']
+      # to override the default packaging platform specified by
+      # beaker-hostgenerator. This should be a comma-separated string with
+      # entries of the format `<host-platform>=<override-platform>`
+      #
+      # @param [Host] host Host whose packaging platform to determine
+      # @return [String] The packaging platform
+      def host_packaging_platform(host)
+        packaging_platform = host[:packaging_platform]
+        if ENV['BEAKER_PACKAGING_PLATFORMS']
+          overrides = Hash[ENV['BEAKER_PACKAGING_PLATFORMS'].split(',').map { |e| e.split('=') }]
+          logger.debug("Found packaging platform overrides: #{overrides}")
+          if overrides[host[:packaging_platform]]
```

Comment:
Shouldn't this be looking up overrides for `host[:platform]`?

Reply:
Yeah, it should -- my aix tests just happened to work out for that specific case anyway. I'll update.